cli/internal/ffi/bindings.h

```diff
@@ -0,0 +1,11 @@
+#include <stdarg.h>
```

Comment:
if these files are generated they probably should not be in the .gitignore

Reply:
Should, or shouldn't be? We have been checking in go proto files so I decided to include it.

Reply:
(Current state seems correct to me.)